python/dolfin/__init__.py

```diff
@@ -31,13 +31,12 @@
 # Import cpp modules
 from .cpp import __version__
 
-from .cpp.common import (Variable, has_debug, has_hdf5, has_scotch,
-                         has_hdf5_parallel, has_mpi4py, has_petsc_complex,
-                         has_petsc4py, has_parmetis, has_slepc, has_slepc4py,
-                         git_commit_hash, DOLFIN_EPS, DOLFIN_PI, TimingType,
-                         timing, timings, list_timings)
+from .cpp.common import Variable, TimingType, timing, timings, list_timings
+from .cpp.common import config
+DOLFIN_EPS = config.DOLFIN_EPS  # FIXME: Remove me
+DOLFIN_PI = config.DOLFIN_PI  # FIXME: Remove me
```

Comment:
These variables need removing (because `dolfin.config.DOLFIN_FOO` is read-only but `dolfin.DOLFIN_FOO` is not any more). Will appear in another PR.

Reply:
Solved in #133